src/middleware/authentication.middleware.ts

```diff
@@ -11,7 +11,8 @@ import {
 export const authentication = (req: Request, res: Response, next: NextFunction): void => {
   try {
     if (!checkUserSignedIn(req.session)) {
-      logger.infoRequest(req, 'User not authenticated, status_code=401, redirecting to sign in page');
+      logger.info("User not authenticated, status_code=401");
```

Comment:
should the logger.info be logger.infoRequest on line 14? it should give us path info and I think request id

Reply:
There is already a `logger.infoRequest` on line 15 I did not want to log the same `req` twice

Reply:
Is just logging the `request.path` and `request.method`, I will push the fix shortly